cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
+yours are different!
+
+However, the plan is to include the following
++ Nomenclature (multiple languages)
++ [Types of Cloth Diapers](#types-of-cloth-diapers)
++ [Materials and Absorbency](#materials-and-absorbency)
++ [Newborn vs One Size](#newborn-vs-one-size)
++ Velcro vs Poppers
++ [Our Choices](#our-choices)
++ Things to Know
++ What causes leaks
+
+## Types of Cloth Diapers
+![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)
+
+I don't want to go into all the different types of cloth diaper just yet as
+there are actually a lot of decent resources online already on this, such as
+https://www.thenappygurus.com/types-of-cloth-nappies.html
+
+The whole of the [Nappy Gurus Help & Advice](https://www.thenappygurus.com/help.html)
+section is extremely well put together and very helfpul if you're starting out.
+
+I'm just leaving this diagram here as a reference point to discuss the
+choices we have made (see below).
+
+## Materials and Absorbency
+
+<img width="664" alt="image" src="https://user-images.githubusercontent.com/4185328/169095781-8c3c8f69-3ae5-4e04-ab39-780f198651a5.png">
+
+Cloth diapers hinge on both their speed of and capacity for absorbency.
+
+The only other thing to mention other than the table above is 'Charcoal Bamboo'.
+This is actually not bamboo at all, it's just a fancy name for microfibre and
+equally unhelpful.
+
+### Absorbency capacity
+The more capacity for absorbency, the more liquid a diaper will hold.
+
+Having said this, every baby is different so whilst some can get away with
+very little absorbency, others are considered 'medium to heavy wetters' and
+will need a lot more absorbency, relying much more heavily on hemp inserts
+(typically used in pocket diapers) or boosters (typically layered into other
+types of diapers to increase absorbency).
+
+### Absorbency speed
+This is important for two reasons:
+1. You need to layer your absorbency so that the layers closer to the baby
+are the ones that absorb liquids the fastest - this will allow that layer to
+absorb the liquids and wick them away to the lower layers which will have more
+storage capacity.
+2. If you have a baby who 'floods'; i.e. releases a large quantity of liquid
+in a short time frame. Here you want the fastest possible absorption to wick
+away the moisture so that you don't get leaks.
+
+## Newborn vs One Size
```

Comment:
Typically the word "versus" is abbreviated to "vs." or "v". https://dictionary.cambridge.org/dictionary/english/versus
```suggestion
## Newborn vs. One Size
```
But _most_ people will understand that this is not "Very Small" ... 
so feel free to ignore. 👍

Reply:
`Sizes` makes sense. I'll update the contents table as well.